src/sentry/issues/search.py

```diff
@@ -219,8 +220,9 @@ def _query_params_for_generic(
         if categories is None:
             logging.error("Category is required in _query_params_for_generic")
             return None
-
         category_ids = {gc.value for gc in categories}
+        if referrer != "api.feedback_index":
+            category_ids.discard(GroupCategory.FEEDBACK.value)
```

Comment:
While we are here can we also apply the inverse: 
If the referrer is feedback, replace any categories and ensure `GroupCategory.FEEDBACK` is present and the only value set.

Reply:
I can follow up with this -- probably want to slightly reorganize how we're doing this.